test/testutility.h

```diff
@@ -103,6 +103,17 @@ private slots:
 
 
     }
+
+    void testVersionOfInstalledBinary()
+    {
+	if( isLinux() ) {
+                QString ver = versionOfInstalledBinary("/home/kf/owncloud.com/buildmirall/bin/owncloud");
```

Comment:
this cannot work. 

Also it is only testing ver is not empty,  not that it is a version.